RHOARDOC-1021 JVM metrics using Jolokia

inoxx03 opened the pull request:
procedure for accessing the hawt.io JVM console on Minishift

Ladicek commented on docs/topics/proc_accessing-jolokia-using-hawtio-on-openshift.adoc:
```diff
@@ -0,0 +1,32 @@
+It allows you to access CPU, storage, and memory usage data collected by JMX over an HTTP bridge using a REST interface and JSON-formatted message payloads, making it suitable for monitoring cloud applications, due to its comparably high speed and low resource-requirements.
+
+
+All Fabric8 container images expose port `8778` by default on the host of your {OpenShiftLocal}.
```
How are Fabric8 images relevant in this context?

inoxx03 replied:
@Ladicek I see, they aren't really:) I will remove the references to Fabric8 images form this section.

Ladicek commented on docs/topics/proc_accessing-jolokia-using-hawtio-on-openshift.adoc:
```diff
@@ -0,0 +1,32 @@
+* the `oc` client uthenticated
+* a Java-based application container running in a project on OpenShift
+* port `8778` exposed in the container image (exposed by default in Fabric8 application images)
+* latest link:https://github.com/jboss-container-images/redhat-openjdk-18-openshift-image/blob/openjdk18-dev/image.yaml[JDK 1.8.0 image^]
```
Maybe don't mention the Fabric8 images at all and instead explain how to enable it on _this_ image?

inoxx03 commented:
@Ladicek  I removed the references to the Fabric8 images and added a steps to edit the yaml template to expose port `8778` and redeploy the application pod. WDYT?
(IIRC, exposing a service on port 8778 using `oc expose` will not work in this context, but i might be missing something)

Ladicek commented on docs/topics/proc_accessing-jolokia-using-hawtio-on-openshift.adoc:
```diff
@@ -4,27 +4,68 @@
++
+[source,bash,options="nowrap",subs="attributes+"]
+--
+oc get pods
```
So if you below edit the deployment config, shouldn't you list the deployment configs here? `oc get dc`

inoxx03 replied:
@Ladicek, yes, that should by all means be mentioned. I'll add it in, thank you for catching this:)

inoxx03 replied:
@Ladicek I've added the step to list the deployment config using `oc get dc`

Ladicek replied:
I think you don't really need to look for the pods. After changing the dc, the pods will have different names anyway. I'd personally get rid of `oc get pods` and just did `oc get dc` (_not_ `oc get dc app-name`). That will tell me the names of all deployment configs and from there, I select the one which I will `oc edit`.

It's not that big of a deal really -- pods managed by a dc are always named after the dc. Your call :-)

inoxx03 commented:
@Ladicek thank you for the feedback, I incorporated your suggestions. Please let me know what you think:)

inoxx03 commented on docs/spring-boot-runtime/master.adoc:
```diff
@@ -15,6 +15,15 @@ include::topics/sb-product-versions.adoc[leveloffset=+2]
 
+== Monitoring
+
+This section contains information about monitoring your {SpringBoot}&ndash;based application running on OpenShift.
```
a. remove double space before variable
b. AFAIK, the `&ndash;` HTML entity is somehow causing the checks to fail

inoxx03 commented:
@rhoads-zach Fixed the build and updated PR.
Had to temporarily disable validation because of hard-coded HTML entiites.
Please feel free to merge:)

rhoads-zach commented:
LGTM, merging.
